griptape/utils/chat.py

```diff
@@ -16,22 +17,39 @@ class Chat:
     prompt_prefix: str = field(default="Q: ", kw_only=True)
     response_prefix: str = field(default="A: ", kw_only=True)
     output_fn: Callable[[str], None] = field(
-        default=Factory(lambda: print), kw_only=True
+        default=lambda x: print(x, end=""), kw_only=True
```

Comment:
What is the reason for removing `Factory`?

Reply:
I'm new to attrs. Factory only seems to accept lambda functions that have no parameter or self. 
`default=Factory(lambda x: print(x))` raises output_fn.<lambda>() missing 1 required positional argument: 'x' when generating __init__

using `default = Factory(lambda: lambda x: print(x,end=""), kw_only=True)` works but is ugly. Is there a better way?

Reply:
Ahh I see, thank you for clarifying. Ok in that case I think we can remove the `Factory`, it looks like we use this pattern [elsewhere](https://github.com/griptape-ai/griptape/blob/main/griptape/drivers/vector/local_vector_store_driver.py#L15). @vasinov any issues with this?